messagequeues/DataCaptureService.cs

```diff
@@ -0,0 +1,87 @@
+﻿
+using System;
+using System.IO;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Tesseract;
+using System.Drawing;
+
+namespace messagequeues;
+
+public class DataCapture
+{
+   public DataCapture()
+   {
+        ConnectionFactory factory = new();
+        factory.Uri = new Uri("amqp://guest:guest@localhost:5672");
+        factory.ClientProvidedName = "DataCaptureService Sender App";
+
+        IConnection cnn = factory.CreateConnection();
+        IModel channel = cnn.CreateModel();
+
+        string exchangeName = "DataCaptureService";
+        string routingKey = "demo-key";
+        string queueName = "DataCaptureServiceQueue";
+
+        channel.ExchangeDeclare(exchangeName, ExchangeType.Direct);
+        channel.QueueDeclare(queueName, false, false, false, null);
+        channel.QueueBind(queueName, exchangeName, routingKey, null);
+
+        byte[] imageBytes = File.ReadAllBytes("test.jpg");

```

Comment:
Task requirement is to "implement Data capture service which will listen to a specific local folder and retrieve documents of some specific format (i.e., PDF) and send to Main processing service through message queue. " which means using something like FileSystemWatcher https://learn.microsoft.com/en-us/dotnet/api/system.io.filesystemwatcher?view=net-8.0